### PR TITLE
docs: Update documentation for `utils.py`

### PR DESCRIPTION
**Pull request by c-dowling**

### CODE REVIEW

File: preprocessing/utils.py [hunk 1]
```diff
@@ -18,18 +15,21 @@ def get_bci_data(self, mat_file):
         self.time = bci_data[1]                  # Vector of the trial time (in ms) relative to target presentation
         self.positionx = bci_data[2]             # X position cursor during feedback
         self.positiony = bci_data[3]             # Y position of cursor during feedback
-        self.SRATE = int(bci_data[4])                 # Sampling rate of EEG recording
-        self.trialData = bci_data[5][0]             # Data structure describing trial level metrics
+        self.SRATE = int(bci_data[4])            # Sampling rate of EEG recording
+        self.trialData = bci_data[5][0]          # Data structure describing trial level metrics
         self.metadata = bci_data[6]              # Participant and session level demographic information
         self.chaninfo = bci_data[7]              # Information about individual EEG channels
```
Comment (RogerAylagas) [inline preprocessing/utils.py:22]: Specify Arguments and Outputs

File: preprocessing/utils.py [hunk 2]
```diff
@@ -18,18 +15,21 @@ def get_bci_data(self, mat_file):
...
 
     def get_num_trials(self):
+        """Returns the number of trials in a given session"""
         return len(self.data[0])
```
Comment (RogerAylagas) [inline preprocessing/utils.py:26]: Specify Arguments and Outputs

File: preprocessing/utils.py [hunk 3]
```diff
@@ -18,18 +15,21 @@ def get_bci_data(self, mat_file):
...
 
     def get_trial_length(self, trial_n):
+        """Returns the length of a given trial"""
         return self.trialData[trial_n][4]
```
Comment (RogerAylagas) [inline preprocessing/utils.py:30]: Specify Arguments and Outputs

File: preprocessing/utils.py [hunk 4]
```diff
@@ -1,8 +1,5 @@
-import pandas as pd
 from scipy.io import loadmat
-import numpy as np
 import os
-import h5py
 
 class Session:
     def __init__(self, file, path):
```
Comment (RogerAylagas) [inline preprocessing/utils.py:5]: TODO: Session documentation. Explain Class. CHeck how this should be done

**Review by RogerAylagas — CHANGES_REQUESTED** [review]
Review the comments in the code and make the requested changes :)

**Comment by c-dowling** [comment]
Just looking at `get_trial_data` and it returns the outputs ("tasknumber, "trialnumber") as 1x1 np.arrays. Should make it return the int/float/bool values instead? Seems like it would be better aesthetically.

**Pull request merged**